api/src/main/java/io/strimzi/api/kafka/model/listener/KafkaListenerAuthenticationOAuth.java

```diff
@@ -138,6 +140,28 @@ public void setUserNameClaim(String userNameClaim) {
         this.userNameClaim = userNameClaim;
     }
 
+    @Description("Configure if access token type check should be skipped. Set this to `true` if authorization server " +
+            "does not include 'typ' claim in JWT token. Defaults to `false`.")
+    @JsonInclude(JsonInclude.Include.NON_DEFAULT)
+    public boolean isSkipTypeCheck() {
```

Comment:
Would `checkAccessTokenType` (defaulting to true) be a better name?

Reply:
Sure, why not. Changed the name to `checkAccessTokenType`.